### ISSUE TITLE
Opening and closing file corupts image

### ISSUE THREAD

**Issue by LMckim** [PHPOffice/PhpSpreadsheet#1222]
When opening an Excel file then writing to a new file the image stamped to each worksheet is corrupted.
### What is the expected behavior?
The image should remain

### What is the current behavior?
The image becomes corrupted

### What are the steps to reproduce?
```php 
$reader = IOFactory::createReaderForFile($path);
$excel = $reader->load($path);
$fileName = "test";
$writer = IOFactory::createWriter($excel,"Xlsx");
$writer->save($fileName.".xlsx");
```

### Which versions of PhpSpreadsheet and PHP are affected?
PHP 7.3.7 PHPSpreadsheet ^1.9

**Comment by darenas31415** [comment]
Hi @LMckim,

I've tested it and the new Excel file still contains the original image.

Please provide a sample file to reproduce the issue, that would help somebody finding the problem.

Thanks

**Comment by oleibman** [comment]
Fixed by PR #1462.
